src/Razor/Microsoft.AspNetCore.Razor.Tools/PublicAPI.Unshipped.txt

```diff
@@ -0,0 +1 @@
+#nullable enable
```

Comment:
Does this project produce shipping API?

Reply:
@SteveSandersonMS and @pranavkm look like the most likely to know for sure but it doesn't look like it to me. This is `rzc`, a tool and not runtime code.

Reply:
Hah - it's in the wrong directory anyway.  Missing `src`